Capstone project

kusiLaw opened the pull request:
This Capstone Project as the following functionalities with the topic "Free Web Boot camp 2022-2023
- Desktop and mobile Version
- The home page have a link in the menu to the about page
- The logo in the header links to the home page
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page
- The section "Main Lecturers "  loads dynamically in JavaScript and mobile by clicking more button.

### Live Video
click here [Loom](https://www.loom.com/share/4b0b9691e4a047d1ba6b0347f9197fca)

kusiLaw committed:
Add linter files

kusiLaw committed:
Add basic files

- add html and css
- add sass files

kusiLaw committed:
update md file

kusiLaw committed:
Add header to page

kusiLaw committed:
Add main program

kusiLaw committed:
updatemain program with join botton

kusiLaw committed:
Add main speaker mobile  section

kusiLaw committed:
Update mobile version responsiveness

- Main program response well
- header layout issue fix

kusiLaw committed:
Add toggle menu event

- menu display mobile item when click
- add event listeners to nav items

kusiLaw committed:
Add then about section of mobile version html structure

kusiLaw committed:
Update mobile about version

- it add sponsor, partners and previous camp section

kusiLaw committed:
Update nav toggle to extend full width

kusiLaw committed:
add desktop navigation bar

kusiLaw committed:
Update header to fit desktop version screen height

kusiLaw committed:
update daily activities

- it makes program responsive for both mobile and desktop
- it scale hover effect to 1.2

kusiLaw committed:
Update program list gap

kusiLaw committed:
Update Lecture section

- it loads all lecturers on desktop version
- it loads all lecturers on window resized to desktop from mobile
- mobile version load only first two lecturers

kusiLaw committed:
Add nav to about desktop version

kusiLaw committed:
Resize logo to suit design

kusiLaw committed:
fix overlay issue

- previously, alumni pictures overlay nav

kusiLaw committed:
Update picture to have same size

kusiLaw committed:
update lecturer load more button functionality

kusiLaw committed:
Fix linter js and html linter error

kusiLaw committed:
Remove saas to pass linter error

- this untrack sass since css linter complain about unknown
use of @use import of @import which is depreciated

kusiLaw committed:
remove sass to pass linter checks

kusiLaw committed:
fix linter error

kusiLaw committed:
Fix bugs

- css linter dose not allow sass and is therefore remove
- fix linter error introduce upon removing sass

kusiLaw committed:
Fix alumni img responsiveness

kusiLaw committed:
Fix mobile home broken link

kusiLaw committed:
Remove partners from about page to match design

kusiLaw committed:
Update README.md

kusiLaw committed:
Update README.md

Add live video  link

kusiLaw committed:
Fix Linter error

kusiLaw committed:
fix logo broken  link

kusiLaw committed:
Add lato font to page

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,273 @@
+                <li class="nav-items ">
+                  <button type="button" class="nav-items-content">
+                    <span>Courses</span>
+                    <img id="pointer" class="" src="./images/arrow_down.png" alt="">
```
- [ ] For accessibility reasons, it would be good to conclude the "alternative" text to all your images so that in case your image fails to render, the users can get a hint of what it was supposed to do. For a refresher on accessibility, kindly look at [this](https://github.com/microverseinc/curriculum-html-css/blob/main/accessibility.md) Microverse learning material on accessibility.

**Kindly do so to all the places where the image tags have no or an empty alt attribute.**

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,273 @@
+        </nav>
+      </div>
+
+      <div class="content">
+        <h1><span>"Hello! sharing World"</span>FREE WEB DEVELOPMENT  BOOTCAMP <br>(2022~2023)</h1>
+        <p class="header-mob-content">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaerat id autem molestiae animi. Nam laborum
+          quaerat quisquam mollitia, amet inventore recusandae velit voluptatum . </p>
+       
+        <h2>2022.10.15~2023.10.15<span>@Valley View University Gh and Live online</span></h2>
+        </div>
```
- [ ] Great job in the styling of this section :clap: . You are left with one small bit. The margin-bottom of this section does not match the overall project look and also the provided Behance design. Kindly increase the margin/ padding bottom for this section and it will look perfect.

**This is how I am seeing your design. I am o 100% zoom level**

![image](https://user-images.githubusercontent.com/86318284/183070562-676416f8-c3a4-431a-9637-0b0ba8ca2c79.png)

**This is the desired design from the Behance design guidelines:**

![image](https://user-images.githubusercontent.com/86318284/183070795-f647c315-4e46-46e3-90e2-27599f38b853.png)

Nemwel-Boniface commented on about.html:
```diff
@@ -0,0 +1,153 @@
+              </ul>
+            </div>
+            <div class="desktop-nav">
+              <div class="desk-logo"><a href=""><img src="./images/logo5.png" alt=""></a></div>
```
- [ ] [OPTIONAL] It would be a good thing to make the logo icon link to the home page. This is normally industry standard and it usually improves the UX of your website.

Nemwel-Boniface reviewed (changes requested):
## Hello @kusiLaw            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done an amazing job with your capstone project :clap: I really enjoyed doing a review of your project and may you continue with the very same spirit even in the next JavaScript module :rocket:  
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)
## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  
Excellent presentation video ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kusiLaw committed:
Fix review bugs

- all img tags has alt name
- welcome screen padding increase to match the design
- logo now links to home page

cwaku reviewed (approved):
Hi @kusiLaw 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 
![](https://media.giphy.com/media/3o7abKhOpu0NwenH3O/giphy.gif)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
